Assets/Scripts/Minigame/DragBlanket.cs

```diff
@@ -104,16 +131,29 @@ private void OnMouseDrag()
                 _rigidbody2D.position = _pos;
             }
 
-            if (_distance >= _winDistance && _counter == 3)
+            if (_standingUp)
             {
-                _tmpUGUI.text = "" + _winMessage;
-                IsDone = true;
+                if (_distance >= _winDistance && _counter == 3)
+                {
+                    _tmpUGUI.text = "" + _winMessage;

```

Comment:
Why add the win message to an empty string? just do
_tmpUGUI.text = _winMessage;